cypress/e2e/privatePaths/render/organizations/members.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { allDevicesForAppLayout } from "@utils/devices";
+import { newExpectationWithClickFlows, newClickFlow } from "@utils/helpers";
+import { runTestsForDevices } from "@utils/index";
+import {
+  shouldFindAndContain,
+  beVisibleAndContain,
+  shouldFindAndCloseModal,
+} from "@utils/sharedTests/sharedFunctionsAndVariables";
+
+const pageName = "Organization page: Members tab";
+const currentPage = "/organizations/testorg?tab=members";
+const loggedIn = true;
+
+const addMemberModalFindAndContains = [
+  {
+    datacy: "add-member-button",
+    text: "Add member",
+  },
+  {
+    datacy: "modal-title",
+    text: "Add member to testorg",
+  },
+];
+
+const editMemberFindAndContains = [
+  {
+    datacy: "edit-button-cypresstesting",
+    text: "Edit",
+  },
+  {
+    datacy: "cypresstesting-member",
+    text: "Member",
+  },
+  {
+    datacy: "cypresstesting-owner",
+    text: "Owner",
+  },
+  {
+    datacy: "cypresstesting-remove",
+    text: "Remove",
+  },
+];
+
+describe(pageName, () => {
+  const tests = [
+    shouldFindAndContain("organization-header", "organizations / testorg"),
+
+    ...addMemberModalFindAndContains.map(test =>
+      test.datacy.includes("button")
+        ? newExpectationWithClickFlows(
+            `should find ${test.text}`,
+            `[data-cy=${test.datacy}]`,
+            beVisibleAndContain(test.text),
+            [newClickFlow(`[data-cy=${test.datacy}]`, [])],
+          )
+        : shouldFindAndContain(test.datacy, test.text),
+    ),
+    shouldFindAndCloseModal("add-member-modal-buttons"),
+
+    ...editMemberFindAndContains.map(test =>
+      test.datacy.includes("button")
+        ? newExpectationWithClickFlows(
+            `should find ${test.text}`,
+            `[data-cy=${test.datacy}]`,
+            beVisibleAndContain(test.text),
+            [newClickFlow(`[data-cy=${test.datacy}]`, [])],
+          )
+        : shouldFindAndContain(test.datacy, test.text),
+    ),
```

Comment:
Should remove button from this array and put it on its own. And then you can just map and use `shouldFindAndContain`